src/main/scala/ie/zalando/fabric/gateway/models/SynchDomain.scala

```diff
@@ -98,7 +99,7 @@ object SynchDomain {
                                   operation: MethodMatch,
                                   allowedRequests: Int,
                                   period: RateLimitPeriod)
-      extends SkipperFilter {
+    extends SkipperFilter {
```

Comment:
Are these format changes generated from scalafmt?

Reply:
They weren't, they were from moving the project